core/src/test/scala/org/apache/spark/sql/delta/ConvertToDeltaScalaSuite.scala

```diff
@@ -21,18 +21,16 @@ import org.apache.spark.sql.types.StructType
 class ConvertToDeltaScalaSuite extends ConvertToDeltaSuiteBase {
   override protected def convertToDelta(
       identifier: String,
-      partitionSchema: Option[String] = None): Unit = {
+      partitionSchema: Option[String] = None, collectStats: Boolean = false): Unit = {
```

Comment:
didn't I comment last time that we wanted this to be `true` by default?

the new default is `true` ... so we want most of our tests to explicitly test for that.

Reply:
thus your new test should be one where it is explicitly set to false.

how many tests break when you set the default to true?

Reply:
Sorry, missed that. There are no tests that break because of this change in this class.